### ISSUE TITLE
Render pretrained model error

### ISSUE THREAD

**Issue by czc2000** [juno181/Ex4DGS#2]
I tried training on the n3d dataset at first and found that the loss was consistently large. Then I downloaded your pre-training model coffee_martini and followed the instructions and found that the rendered image is almost all black, can you help me?
Here are the results I got ,the first is the render image and the second is gt.
![0_render](https://github.com/user-attachments/assets/ff2cb70f-ec3d-4485-9856-6583c20a4daf)
![0](https://github.com/user-attachments/assets/92664319-4550-4c35-bc93-d385424e730e)

**Comment by juno181** [comment]
Hi, thanks for reporting this issue! I just found out that the latest version of glm is not compatible with my code. Version [v1.0.1 of glm](https://github.com/g-truc/glm/tree/1.0.1) is compatible.

To resolve this, please replace `submodules/diff_gaussian_rasterization_df/third_party/glm` with v1.0.1 or re-clone this repository to ensure compatibility. Thank you!

**Comment by czc2000** [comment]
Thanks your help!
